articles/machine-learning/how-to-auto-train-image-models.md

```diff
@@ -359,14 +359,43 @@ In individual trials, you directly control the model architecture and hyperparam
 
+With the new AutoML on Pipelines feature (preview), you can additionally use any image classification model from the [HuggingFace Hub](https://huggingface.co/models?pipeline_tag=image-classification&library=transformers) which are part of the transformers library (such as microsoft/beit-base-patch16-224), as well as any object detection or instance segmentation model from the [MMDetection Version 2.28.2 Model Zoo](https://mmdetection.readthedocs.io/en/v2.28.2/model_zoo.html) (such as atss_r50_fpn_1x_coco). Using any HuggingFace or MMDetection model will trigger runs using pipeline components. If both legacy and HuggingFace/MMdetection models are used, all runs/trials will be triggered using components. Curated models from HuggingFace and MMDetection are provided in the azureml-staging registry. These curated models have been tested thoroughly and use default hyperparameters selected from extensive benchmarking to ensure effective training. The table below summarizes these curated models
+
+Task |  model architectures
+---|---------
+Image classification<br> (multi-class and multi-label)| [**microsoft/beit-base-patch16-224-pt22k-ft22k**](https://ml.azure.com/registries/azureml-preview/models/microsoft-beit-base-patch16-224-pt22k-ft22k/version/1): The BEiT model is a Vision Transformer (ViT), which is a transformer encoder model (BERT-like). <br> [**google/vit-base-patch16-224**](https://ml.azure.com/registries/azureml-preview/models/google-vit-base-patch16-224/version/1): The Vision Transformer (ViT) is a transformer encoder model (BERT-like) pretrained on a large collection of images in a supervised fashion, namely ImageNet-21k, at a resolution of 224x224 pixels. <br> [**facebook/deit-base-patch16-224**](https://ml.azure.com/registries/azureml-preview/models/facebook-deit-base-patch16-224/version/1): This model is actually a more efficiently trained Vision Transformer (ViT). <br> [**microsoft/swinv2-base-patch4-window12-192-22k**](https://ml.azure.com/registries/azureml-preview/models/microsoft-swinv2-base-patch4-window12-192-22k/version/1): The Swin Transformer is a type of Vision Transformer.
+Object Detection | TODO
+Instance Segmentation | TODO
+
+You can also get a list of the curated models for a given task using the Python SDK:
+```
+credential = DefaultAzureCredential()
+ml_client = MLClient(credential, registry_name="azureml-staging")
+
+models = ml_client.models.list()
+classification_models = []
+for model in models:
+    model = ml_client.models.get(model.name, label="latest")
+    if model.tags['task'] == 'image-classification': # choose an image task
+        classification_models.append(model.name)
+
+classification_models
+```
+output:
+```
+['google-vit-base-patch16-224',
+ 'microsoft-swinv2-base-patch4-window12-192-22k',
+ 'facebook-deit-base-patch16-224',
+ 'microsoft-beit-base-patch16-224-pt22k-ft22k']
+```
```

Comment:
I would move here the phrase: "Using any HuggingFace or MMDetection model will trigger runs using pipeline components. If both legacy and HuggingFace/MMdetection models are used, all runs/trials will be triggered using components."

The users should not care much about this part, at least not until they already read the above